### PR TITLE
refactor: rename content information fileds

### PR DESCRIPTION
**Pull request by cblk**

### CODE REVIEW

File: simpleContentNft [hunk 1]
```diff
@@ -26,10 +26,10 @@ contract simplecontentNft is Context, ERC165, IERC721, IERC721Metadata {
 
     /*** Content-NFT DATA TYPES ***/
     struct ContentInformtaion {
-        string[] _productInfo; // product information of the content: text or URI
-        string[] _contractDocuments; // contract documents: text or URI
-        uint256[] _copyrightFeeRatio; // copyright fee ratio of NFT
-        bool _allowSecondaryMerket; // permission of secondary market sale :Just reference information
+        string[] _info; // product information of the content: text or URI
+        string[] _Documents; // contract documents: text or URI
```
Comment (YErikOhtake) [inline simpleContentNft]: Property names must begin with a lowercase letter.
_Documents -> _documents

File: simpleContentNft [hunk 2]
```diff
@@ -26,10 +26,10 @@ contract simplecontentNft is Context, ERC165, IERC721, IERC721Metadata {
 
     /*** Content-NFT DATA TYPES ***/
     struct ContentInformtaion {
-        string[] _productInfo; // product information of the content: text or URI
-        string[] _contractDocuments; // contract documents: text or URI
-        uint256[] _copyrightFeeRatio; // copyright fee ratio of NFT
-        bool _allowSecondaryMerket; // permission of secondary market sale :Just reference information
+        string[] _info; // product information of the content: text or URI
+        string[] _Documents; // contract documents: text or URI
+        uint256[] _Ratio; // copyright fee ratio of NFT
```
Comment (YErikOhtake) [inline simpleContentNft]: Property names must begin with a lowercase letter.
_Ratio -> _ratio

File: simpleContentNft [hunk 3]
```diff
@@ -126,10 +126,10 @@ contract simplecontentNft is Context, ERC165, IERC721, IERC721Metadata {
         bool _asm
     ) {
 
-        string[] memory pi = _contents[tokenId]._productInfo;
-        string[] memory cds = _contents[tokenId]._contractDocuments;
-        uint256[] memory crfr = _contents[tokenId]._copyrightFeeRatio;
-        bool asm = _contents[tokenId]._allowSecondaryMerket;
+        string[] memory pi = _contents[tokenId]._info;
```
Comment (YErikOhtake) [inline simpleContentNft]: The variable name "pi" does not indicate the entity.
The variable name should be "info".
"cds", "crfr", and "asm" are also the same review comments.

**Review by YErikOhtake — CHANGES_REQUESTED** [review]
I have reviewed. Please correct the source.

File: simpleContentNft (hunk 1 above)
Comment (cblk) [reply simpleContentNft]: ok

File: simpleContentNft (hunk 2 above)
Comment (cblk) [reply simpleContentNft]: ok

**Review by YErikOhtake — APPROVED** [review]

**Comment by KojiMachi** [comment]
Thanks for your cooperation.

**Pull request merged**